proofpoint-threat-response/info.json

```diff
@@ -0,0 +1,883 @@
+{
+  "name": "proofpoint-threat-response",
+  "label": "Proofpoint Threat Response",
+  "version": "1.0.0",
+  "description": "Proofpoint Threat Response connector is used to orchestrate and automate incident response.",
```

Comment:
Please update description with -  `Proofpoint Threat Response is a solution designed to help organizations manage and respond to cybersecurity threats. It provides tools and features to identify, investigate, and remediate security incidents.`